Content/Packets/UpdateSyncDataPacket.cs

```diff
@@ -0,0 +1,51 @@
+﻿using NetEasy;
+using System;
+
+namespace StarlightRiver.Content.Packets
+{
+	/// <summary>
+	/// Used to update the data storage for when they are changed through client dialog, like glass weaver's dialog.
+	/// </summary>
+	[Serializable]
+	public class UpdateSyncDataPacket : Module
```

Comment:
So, this packet has duplicate functionality from setting NPC.netUpdate = true for a boss NPC. The only places this has different functionality is if you are sending more packets then netSpam lets you send normally or if this is being used to sync non boss NPCs with extra ai fields (which should be using SendExtraAI and ReceiveExtraAI if that's the case)

So if this is necessary to bypass the netSpam limits add that to the comments

Reply:
You're completely correct! Sorry for that. I removed UpdateSyncDataPacket, and its now using NPC.netUpdate = true, which also works!